src/store/aggregator/sagas/withdrawTokens.ts

```diff
@@ -29,6 +32,8 @@ export function* withdrawTokens({ payload }: AggregatorActions['submit']) {
   const massetContract = yield* select(massetContractSelector);
   const account = yield* select(accountSelector);
   const { destinationChain, receiveAddress, sendAmount } = payload;
+  const penaltyData = yield* select(incentivesSelector);
+  const penaltyAmount = Number(penaltyData?.amount ?? 0);
```

Comment:
```suggestion

```

Reply:
I don't understand this comment.

Reply:
If you open it in IDE you will. `penaltyAmount` is not used anywhere.